.github/workflows/check-tls-expiry.yml

```
@@ -13,53 +13,24 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v4
 
-      - name: List Omnistrate Instances
-        id: list_instances
-        uses: ./.github/actions/list-omnistrate-instances
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)